### DISCUSSION TITLE
SSH Key-Based Authentication through a Proxy on a Windows 10 Client

### DISCUSSION THREAD

**Discussion by giscus** [benmepham/benmepham.github.io#8] [Blog Comments]
# SSH Key-Based Authentication through a Proxy on a Windows 10 Client

I wanted to connect to one of my University’s Linux computers to compile and run some code. However, in order to do this, I needed to connect through an SSH proxy. To simplify this process, I also wanted to set up SSH key-based authentication. This post will document how I set this up on a Windows 10 client.

https://bjm.me.uk/blog/windows10-ssh-key-proxy/

**Comment by kalulas** [comment]
Thank you for sharing! This really helps
